client/dashboard/task-list/tasks/payments/methods.js

```diff
@@ -71,6 +72,34 @@ export function getPaymentMethods( {
 				options.woocommerce_stripe_settings.enabled === 'yes',
 			optionName: 'woocommerce_stripe_settings',
 		},
+		{
+			key: 'wcpay',
+			title: __( 'WooCommerce Payments', 'woocommerce-admin' ),
+			content: (
+				<Fragment>
+					{ __(
+						'Accept credit card payments the easy way! No setup fees. No ' +
+							'monthly fees. Just 2.9% + $0.30 per transaction ' +
+							'on U.S.-issued cards',
+						'woocommerce-admin'
+					) }
+					{ __(
+						'By clicking "Setup," you agree to the Terms of Service',
+						'woocommerce-admin'
+					) }
+				</Fragment>
+			),
+			before: <img src={ wcAssetUrl + 'images/wcpay.png' } alt="" />,
+			visible: [ 'US' ].includes( countryCode ) && ! hasCbdIndustry,
+			plugins: [ 'woocommerce-payments' ],
+			container: <WCPay />,
+			isConfigured: activePlugins.includes( 'woocommerce-payments' ),
```

Comment:
So this will show that they payment method is configured if the plugin is installed... is there some other signal that we can use from wc-pay to verify that all is connected and ready to accept payments?

Reply:
I noticed that too today. Working on it!

Reply:
This will likely require a change in woocommerce payments - looking into it

Reply:
See https://github.com/woocommerce/woocommerce-admin/pull/3978#issuecomment-605204413